hack/dev-env/start-e2e.sh

```
@@ -49,25 +51,25 @@ getExternalLoadBalancerIP() {
 export MANAGED_AGENT_REDIS_ADDR="$EXTERNAL_IP:6379"
 
 # Get hostname of agent-autonomous redis
 K8S_CONTEXT="--context=vcluster-agent-autonomous"
-K8S_NAMESPACE="-n argocd"
+K8S_NAMESPACE="-n ${ARGOCD_AUTONOMOUS_NAMESPACE}"
 getExternalLoadBalancerIP "argocd-redis"
 export AUTONOMOUS_AGENT_REDIS_ADDR="$EXTERNAL_IP:6379"
 
 
-export REDIS_PASSWORD=$(kubectl get secret argocd-redis --context=vcluster-agent-managed $K8S_NAMESPACE -o jsonpath='{.data.auth}' | base64 --decode)
+export REDIS_PASSWORD=$(kubectl get secret argocd-redis --context=vcluster-agent-managed -n ${ARGOCD_MANAGED_NAMESPACE} -o jsonpath='{.data.auth}' | base64 --decode)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)